build/build-image/cache/cloudbuild.yaml

```
@@ -0,0 +1,146 @@
+---
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+
+  - name: 'gcr.io/cloud-builders/docker'
+    id: build_base_image
+    args:
+      - 'build'
+      - '--tag=us-docker.pkg.dev/$PROJECT_ID/ci/cache:${_VERSION}'
+      - '--tag=us-docker.pkg.dev/$PROJECT_ID/ci/cache:latest'
+      - '--file=Dockerfile-base'
+      - '.'
+
+  - name: 'us-docker.pkg.dev/$PROJECT_ID/ci/save_cache'
```

```suggestion
  - name: 'gcr.io/cloud-builders/docker'
```

Since this is building the docker container, so it needs to use docker as the build step container.